docs/kubernetes/deploy-your-application-to-google-cloud-on-your-ci-cd.md

```diff
@@ -0,0 +1,141 @@
+# Deploy your application to Google Cloud on your CI/CD
+This tutorial describes how to set up your CI/CD to deploy your application to Google Cloud.
+
+For this example, we will be using Jenkins as our CI.
+
+## Set up Google Cloud
+
+First configure your account to allow deploy using APIs.
+
+### Enable API
```

Comment:
remove whole section since it is setup in https://github.com/shopsys/shopsys/blob/mc-google-cloud-infrastructure/docs/kubernetes/set-up-google-cloud.md